### PR TITLE
Feature/specify steamdir

### PR DESCRIPTION
**Pull request by triffid**
I don't have anything in my default steam app directory (~/.steam/steam/steamapps), this patch allows me to specify other locations to look.

Ideally we can grab the list of locations from steam settings somewhere.

### CODE REVIEW

File: protonsh.sh [hunk 1]
```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
 
+if [ -r ~/.config/protonsh ]
+then
+	source ~/.config/protonsh
+fi
+
 shell="${SHELL:-/bin/bash}"
+steamdir="${STEAMDIR:-$HOME/.steam/steam/SteamApps}"
```
Comment (414n) [inline protonsh.sh]: I'd prefer a more specific variable name pair, like `steamAppsDir`/`STEAM_APPS_DIR`.
Do you mind changing it?

**Comment by 414n** [comment]
Hi @triffid,

thanks for your contribution. I just left a single review note about a variable name.
After that is dealt with, I'll gladly merge this! :+1:

**Comment by triffid** [comment]
Hmm I can't see your note anywhere, did github eat it or is its visibility private or something?

**Review by 414n — CHANGES_REQUESTED** [review]

**Comment by 414n** [comment]
> Hmm I can't see your note anywhere, did github eat it or is its visibility private or something?

Sorry, my fault. It seems like it was kept there as a draft until now. Do you see it?

**Comment by triffid** [comment]
There, how's this?

**Comment by 414n** [comment]
Perfect, thank you very much!

**Pull request merged**